tests/perf/cypress/e2e/Graphload.cy.ts

```diff
@@ -0,0 +1,74 @@
+describe('graphload', () => {
```

Comment:
nitpick: can this file be named `Graphload.spec.ts`? Makes it more consistent with other frontend test files.

Reply:
I will add the change.